examples/toolkit/scripts/fetch.py

```
@@ -0,0 +1,75 @@
+
+    if sort not in SORTS:
+        print(f"Invalid sort: {sort}. Choose from: {', '.join(SORTS)}", file=sys.stderr)
+        sys.exit(1)
+
+    limit = max(1, min(25, limit))
+
+    try:
+        result = fetch(subreddit, sort, limit)
+        json.dump(result, sys.stdout, indent=2)
+        print()
+    except Exception as e:
```

Ruff (0.15.5)

[warning] Do not catch blind exception: `Exception`

(BLE001)